src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/UnoQuickStart.sln

```diff
@@ -185,6 +185,7 @@ Global
 		{$guid1$}.Release|x86.Build.0 = Release|x86
 		{$guid1$}.Release|x86.Deploy.0 = Release|x86
 
+#//#if (skia-wpf)
```

Comment:
It feels like only the first block should be unconditionally removed, there's probably no need to make it conditional to skia-wpf.

Reply:
https://github.com/unoplatform/uno/commit/702900e33c1705a77d615c48be661405f3fbb375 remove first block and condition